meta/empty_message.test.yaml

```diff
@@ -0,0 +1,8 @@
+rules:
+- id: empty-message
+  # ruleid: empty-message
+  message: |
+  languages: [yaml]
```

Comment:
This is a bit confusing. The .test.yaml could be any kind of rule, like [c]. Using [yaml] here and having the test almost identical to the rule is confusing.